src/components/layout/Navigation.tsx

```diff
@@ -74,14 +75,24 @@ const navItems: NavItem[] = [
  */
 export const Navigation: FC<{}> = () => {
   const collapseWallet = useCloseAccount();
+  const selectedMasset = useSelectedMasset();
   const { pathname } = useLocation();
   const items: (NavItem & { active: boolean })[] = useMemo(
     () =>
       navItems.map(item => ({
         ...item,
-        active: !!(item?.path && pathname.startsWith(item.path)),
+        path:
+          item.path === '/earn' || item.path === '/faq'
```

Comment:
A better solution now we have react-router: use this component instead: https://reactrouter.com/web/api/NavLink and you won't need to set the `active` prop.

Reply:
Also this is probably neater if you create an array or set of items that _do_ use the mAsset, e.g. all earn paths won't have it